packages/core/src/trigger_api/keep_fresh.ts

```diff
@@ -40,11 +41,37 @@ export function keepFresh<Params>(
   }
 ): void;
 
+export function keepFresh<Params>(
```

Comment:
🤓 as I understand, we can add `enabled?: Store<boolean>` to existent overloads instead of creating new. Am I wrong?

Reply:
You are right.
Fixed it!